build/vcpkg_local_ports/libdatadog/portfile.cmake

```diff
@@ -0,0 +1,48 @@
+set(LIBDATADOG_VERSION ${VERSION})
+
+if(#TARGET_TRIPLET STREQUAL "x64-windows" OR
+   TARGET_TRIPLET STREQUAL "x64-windows-static")
+    set(PLATFORM "x64")
+    set(LIBDATADOG_HASH "bd3061cf7811a3a10601c5f5e4be871c9b806a8afac9343802b47878e997a2596426f030b903dfb6aa6ae0961d41c0ad844295d2eb728ee3552db3cfc924a305")
+elseif(TARGET_TRIPLET STREQUAL "x86-windows" OR
+       TARGET_TRIPLET STREQUAL "x86-windows-static")
+    set(PLATFORM "x86")
+    set(LIBDATADOG_HASH "e81eb5881a06186111e03d012b10236a32e7a2d1c2d2c0e4a7a19646e280923d6b0a4b2e6ebb588fa358e3304edf5c8fb0ff37383361f577fbfd9ad75dc27aab")
+else()
+    message(FATAL_ERROR "Unsupported triplet: ${TARGET_TRIPLET}")
+endif()
+
+# Define the version and download URL for the prebuilt binaries
+set(LIBDATADOG_FILENAME "libdatadog-${PLATFORM}-windows")
+set(LIBDATADOG_ARTIFACT "${LIBDATADOG_FILENAME}.zip")
+set(LIBDATADOG_URL "https://github.com/gleocadie/libdatadog/releases/download/v${LIBDATADOG_VERSION}/${LIBDATADOG_ARTIFACT}")
```

Comment:
This will be remove once the zip files are pushed in libdatadog release page for the v14.3.1